com.playeveryware.eos/Runtime/Core/EOSManager.cs

```diff
@@ -474,49 +471,39 @@ private Result InitializePlatformInterface()
             //-------------------------------------------------------------------------
             private PlatformInterface CreatePlatformInterface()
             {
-                EOSConfig configData = Config.Get<EOSConfig>();
+                PlatformConfig platformConfig = PlatformManager.GetPlatformConfig();
+                ProductConfig productConfig = Config.Get<ProductConfig>();
+
                 IPlatformSpecifics platformSpecifics = EOSManagerPlatformSpecificsSingleton.Instance;
 
-                EOSCreateOptions platformOptions = new EOSCreateOptions();
+                EOSCreateOptions platformOptions = new();
 
-                
                 platformOptions.options.CacheDirectory = platformSpecifics.GetTempDir();
-                platformOptions.options.IsServer = configData.isServer;
+                platformOptions.options.IsServer = platformConfig.isServer;
                 platformOptions.options.Flags =
 #if UNITY_EDITOR
                     PlatformFlags.LoadingInEditor;
 #else
                     configData.platformOptionsFlags.Unwrap();
 #endif
-                if (EOSClientCredentials.IsEncryptionKeyValid(configData.encryptionKey))
-                {
-                    platformOptions.options.EncryptionKey = configData.encryptionKey;
```

Comment:
We still need to be setting this. Log an error if the encryption key is invalid.

Reply:
Fixed, marking as resolved.